Node/core/lib/locale/es/BotBuilder.json

```diff
@@ -1,16 +1,16 @@
 {
     "default_text": "No se pudo reconocer su respuesta. Por favor vuelva a intentar.",
     "default_number": "No se pudo reconocer el texto como número. Vuelva a intentar sólo con números.",
-    "default_confirm": "No se pudo reconocer la confirmación. Por favor responda con 'si' o 'no'.",
+    "default_confirm": "No se pudo reconocer la confirmación. Por favor responda con 'sí' o 'no'.",
     "default_choice": "No se pudo reconocer su respuesta. Por favor seleccione una opción de la lista.",
     "default_time": "No se pudo reconocer el formato de la fecha. Por favor vuelva a intentar (MM/DD/YYYY HH:MM:SS).",
     "default_file": "No se pudo recibir el archivo. Por favor vuelva a intentar.",
     "default_error": "Oops. Ocurrió un problema. Por favor vuelva a intentar.",
     "list_or": " o ",
     "list_or_more": ", o ",
-    "confirm_yes": "si",
+    "confirm_yes": "sí|si",
```

Comment:
*Attention*: `confirm_yes` is used as literal in confirm label, so `sí|si` will be printed, as can be seen ![here](http://i.imgur.com/g1RT2VB.png)